error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/AssertJEnumerableRulesTestOutput.java

```diff
@@ -29,4 +30,9 @@ ImmutableSet<EnumerableAssert<?, Integer>> testEnumerableAssertHasSameSizeAs() {
         assertThat(ImmutableSet.of(3)).hasSameSizeAs(ImmutableSet.of(4)),
         assertThat(ImmutableSet.of(5)).hasSameSizeAs(new Integer[0]));
   }
+
+  ImmutableSet<EnumerableAssert<?, Integer>> testEnumerableAssertMapHasSameSizeAs() {
+    return ImmutableSet.of(
```

Comment:
If we add only one test case, we don't need to wrap the expression in `ImmutableSet.of` 😄.

Reply:
Fair enough, stupid copy pasta, but before making it nice I want to have the right way forward :)

Reply:
The current way is definitely not wrong, but just to mention this; another valid approach would be to first create a GH issue such that we can have a discussion there 😉.

Reply:
😅  I'm less used to the GH issue way ;)